examples/stripe-integration/app/utils/stripe.server.tsx

```diff
@@ -0,0 +1,20 @@
+import initStripe from "stripe";
+
+export const getStripeSession = async (priceId: string): Promise<string> => {
+  const stripe = initStripe(process.env.STRIPE_SECRET_KEY);
+  const lineItems = [
+    {
+      price: priceId,
+      quantity: 1
+    }
+  ];
+  const session = await stripe.checkout.sessions.create({
+    mode: "payment",
+    payment_method_types: ["card"],
+    line_items: lineItems,
+    success_url: "http://localhost:3000/payment/success",
+    cancel_url: "http://localhost:3000/payment/cancelled"
```

Comment:
In the real world, you wouldn't use `http://localhost:3000`. Could you instead get the domain from the `request`? Similar to what I do here: https://github.com/kentcdodds/kentcdodds.com/blob/ebb36d82009685e14da3d4b5d0ce4d577ed09c63/app/utils/misc.tsx#L229-L237

Reply:
that's good idea, just stole it 😁